src/main/java/seedu/modtracker/Parser.java

```diff
@@ -10,21 +10,22 @@
      * Parses user inputs.
      *
      * @param input user input
-     * @param t modlist
+     * @param modules modlist
      * @param name name entered by the user
      */
-    public static void parse(String input, ModuleList t, String name) {
+    public void parse(String input, ModuleList modules, String name) {
```

Comment:
Maybe it would be clearer for the instance of ModuleList to be called modList instead? It might get confused with module.